astroquery/heasarc/core.py

```diff
@@ -15,6 +16,14 @@
 __all__ = ['Heasarc', 'HeasarcClass']
 
 
+def Table_read(*args, **kwargs):
+    # why does if commons.ASTROPY_LT_5_0 not work on Windows?
```

Comment:
Ok, sorry, I had a problem with some recent `minversion` elsewhere, so I misinterpreted this.

So you suggest I add `ASTROPY_LT_5_1` in `utils`? It will not be satisfied before 5.1 release, right? Is it ok that the `devastropy` test will fail?

Reply:
I changed as suggested and it does fail astropy dev, as expected.

Reply:
No, I think we better hack it around temporarily until the 5.1 release is out.

As I see this is the only remaining issue, and that there are a few merge commits, so I go ahead, rebase the PR and a commit that address this variable and we're good to go. Ultimately we better fix the unit parsings as the issue came up recently in #2333, too